qtconsole/frontend_widget.py

```diff
@@ -344,7 +340,7 @@ def _insert_continuation_prompt(self, cursor):
         """ Reimplemented for auto-indentation.
         """
         super(FrontendWidget, self)._insert_continuation_prompt(cursor)
-        cursor.insertText(' ' * self._input_splitter.indent_spaces)
+        cursor.insertText(' ' * 4)
```

Comment:
The is_complete_reply does have an 'indent' field to provide a hint for this. I suspect this method is also called when you Ctrl+Enter to force a newline, so always inserting an indent is probably not right.